src/collections/_documentation/learn/search.md

```diff
@@ -132,3 +132,17 @@ The following tokens are reserved and known to Sentry:
   `lastSeen:-2d`
 
 Additionally you can use any tag you’ve specified as a token.
+
+## Saving Searches
+
+You have the ability to save a search by performing the search, clicking the dropdown arraow next to the Current Search, and then "Save Currrent Search".
+
+[{% asset save-current-search.png %}]({% asset save-current-search.png @path %})
```

Comment:
The part in brackets should be an `alt` tag. As coded, this will dump an entire `img` tag inside the alt attribute.